styling/headerStyles.scss

```diff
@@ -1,29 +1,73 @@
-@import url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
 @import "variables";
 
 .header {
   display: flex;
   align-items: center;
   justify-content: center;
-  background-color: $secondary-color;
+  font-family: $font-family;
 
-  #KDSlogo {
-    width: 20em;
-    margin-right: 2em;
+  @media screen and (max-width: $break-small) {
+    flex-direction: column;
+    width: 100%;
+  }
 
+  #KDSlogo {
+    width: 10em;
+    margin-right: 5%;
     @media screen and (max-width: $break-small) {
-      width: 10em;
+      width: 5em;
       margin-right: 1em;
     }
   }
-  #KDSheader {
-    width: 42%;
-    font-size: 4em;
-    color: $primary-color;
+  h2 {
```

Comment:
here you are styling h2, h3 ++ these should be general styles in styles.scss, if there is something specific with the titles used in the header you should possibly use className instead.